### PR TITLE
feat: Landing page created with four sections

### PR DESCRIPTION
**Pull request by MohannadSabaa**
- building Landing page
    - Hero section (background with title)
    - therapy intro section (side image, 3 information boxes about therapy)
    - client section (image with a button to sign up)
    - therapist section ( image with a button to join)

**NOTE: HERO section background will be replaced soon Inshallah ...**

### CODE REVIEW

File: src/pages/landingPage/styles.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+/* import vector from '../../../public/vector2.jpg' */
```
Comment (ahmedmurtaja) [inline src/pages/landingPage/styles.ts]: Remove commented code

File: src/pages/landingPage/index.tsx [hunk 2]
```diff
@@ -1,13 +1,87 @@
...
   
 
   return (
-    <Container>
-     
-      LandingPage
-    </Container>
+    <>
+      <Container sx={hero}> 
+      <h1 className='way'>WAY </h1>
```
Comment (ahmedmurtaja) [inline src/pages/landingPage/index.tsx]: Use MUI Typography instead of p tags and h1

**Review by ahmedmurtaja — CHANGES_REQUESTED** [review]

File: src/pages/landingPage/styles.ts (hunk 1 above)
Comment (MohannadSabaa) [reply src/pages/landingPage/styles.ts]: done

File: src/pages/landingPage/index.tsx (hunk 2 above)
Comment (MohannadSabaa) [reply src/pages/landingPage/index.tsx]: done

**Review by ahmedmurtaja — APPROVED** [review]

**Review by YasserAmro0 — APPROVED** [review]

**Pull request merged**